<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetectionView</class>
 <widget class="QWidget" name="DetectionView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>340</y>
     <width>151</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Trafic Light </string>
   </property>
  </widget>
  <widget class="QPushButton" name="configButton">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>640</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: gray;</string>
   </property>
   <property name="text">
    <string>Configuration</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changeCameraButton">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>680</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Switch Camera</string>
   </property>
  </widget>
  <widget class="QLabel" name="trafficLightStatus">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>360</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>GREEN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>430</y>
     <width>181</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Cars Detected Today</string>
   </property>
  </widget>
  <widget class="QLabel" name="carsDetected">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>450</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>510</y>
     <width>181</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Violations Detected</string>
   </property>
  </widget>
  <widget class="QLabel" name="violationsDetected">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>530</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>1201</x>
     <y>719</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: red;
color: white</string>
   </property>
   <property name="text">
    <string>Stop System</string>
   </property>
  </widget>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1191</width>
     <height>771</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="trafficLight">
   <property name="geometry">
    <rect>
     <x>1330</x>
     <y>330</y>
     <width>28</width>
     <height>64</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>610</y>
     <width>181</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Maintenance</string>
   </property>
  </widget>
  <widget class="QLabel" name="day">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>20</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MONDAY</string>
   </property>
  </widget>
  <widget class="QLabel" name="time">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>40</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>7:02 PM</string>
   </property>
  </widget>
  <widget class="QLabel" name="date">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>70</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>September 9, 2019</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
